docs/data-sources/system.md

```diff
@@ -7,69 +7,90 @@ description: |-
-- `inb_mgmt_addr_gateway6` - (Optional) In-band management IPv6 Gateway.
-- `last_reset_reason`- (Optional) Last reset reason for this system.
-- `lc_own` - (Optional) A value that indicates how this object was created. For internal use only.
-- `mod_ts` - (Optional) The time when this object was last modified.
-- `mode` - (Optional) Specifies if this system is configured in standalone mode or HA pair.
-- `mon_pol_dn` - (Optional) The monitoring policy attached to this observable object.
-- `name` - (Optional) The system name.
-- `oob_mgmt_addr` - (Optional) The out-of-band management IPv4 address.
-- `oob_mgmt_addr6` - (Optional) The out-of-band management IPv6 address.
-- `oob_mgmt_addr6_mask` - (Optional) The out-of-band management IPv6 address Mask.
-- `oob_mgmt_addr_mask` - (Optional) The out-of-band management IP address Mask.
-- `oob_mgmt_gateway` - (Optional) Out-of-band management IP Gateway.
-- `oob_mgmt_gateway6` - (Optional) Out-of-band management IPv6 Gateway
-- `rl_oper_pod_id` - (Optional) Operational POD Idenitfier for RL Pod Redundancy.
-- `rl_routable_mode` -  (Optional) Is Remote-Leaf Routable.
-- `serial` - (Optional) Serial Number of the system.
-- `state` - (Optional) Operational state of this system.
-- `system_uptime` - (Optional) The time (in seconds) since the system was booted.
-- `tep_pool` - (Optional) Tep-Pool for this system
-- `unicast_xr_ep_learn_disable` - (Optional) Disable xrLeanrs.
-- `version` - (Optional) The version of the compatibility catalog.
-- `virtual_mode` - (Optional) Virtual mode of system.
+## Schema ##
+
+### Required ###
+
+* `system_id` (id) - (string) The identifier of the System object.
+* `pod_id` (podId) - (string) POD Idenitfier.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the System object.
+* `address` (address) - (string) The IP address of the system.
+* `bootstrap_state` (bootstrapState) - (string) Bootstrap state of this system.
+* `cluster_time_diff` (clusterTimeDiff) - (string) Difference in cluster time from local time for this system.
+* `control_plane_mtu` (controlPlaneMTU) - (string) MTU for control plane (SUP-originated) packets.
+* `current_time` (currentTime) - (string) The current time on this system.
+* `enforce_subnet_check` (enforceSubnetCheck) - (string) Enforce subnet check on all VRFs.
+* `etep_addr` (etepAddr) - (string) The External TEP IP address of this PoD.
+* `fabric_domain` (fabricDomain) - (string) Fabric domain of this node.
+* `fabric_id` (fabricId) - (string) The latest system health score. Use the navigation bar at the top right of the table to select which health level to view.
+* `fabric_mac` (fabricMAC) - (string) 
+* `inb_mgmt_addr` (inbMgmtAddr) - (string) The in-band management IPv4 address.
+* `inb_mgmt_addr6` (inbMgmtAddr6) - (string) In-band management IPv6 address.
+* `inb_mgmt_addr6_mask` (inbMgmtAddr6Mask) - (string) In-band management IPv6 address Mask.
```

Comment:
Some sentences start with The and other not, perhaps from consistency make it similar?